PowerEditor/installer/nativeLang/polish.xml

```diff
@@ -104,9 +104,11 @@ Translation note:
 					<Item id="41003" name="Za&amp;mknij"/>
 					<Item id="41004" name="Zamknij &amp;wszystko"/>
 					<Item id="41005" name="Zamknij wszystko prócz bieżącego pliku"/>
+					<Item CMDID="41026" name="Zamknij wszystko oprócz przypiętych"/>
```

Comment:
It should be "id" instead of "CMDID".

Reply:
`<Item id=...`
instead of
`<Item CMDID=...`

Reply:
Then add the entry `<Item CMDID="41026" name="Zamknij wszystko oprócz przypiętych"/>` in `<TabBar>`